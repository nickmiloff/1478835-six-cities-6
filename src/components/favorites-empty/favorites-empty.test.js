import React from 'react';
import {render} from '@testing-library/react';

import FavoritesEmpty from './favorites-empty';

describe(`FavoritesEmpty component test`, () => {
  it(`'FavoritesEmpty component' should render correctly`, () => {
    const {container} = render(<FavoritesEmpty />);

    expect(container).toMatchSnapshot();
  });
});
