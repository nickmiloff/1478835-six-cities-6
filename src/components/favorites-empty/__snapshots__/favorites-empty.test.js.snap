// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritesEmpty component test 'FavoritesEmpty component' should render correctly 1`] = `
<div>
  <section
    class="favorites favorites--empty"
  >
    <h1
      class="visually-hidden"
    >
      Favorites (empty)
    </h1>
    <div
      class="favorites__status-wrapper"
    >
      <b
        class="favorites__status"
      >
        Nothing yet saved.
      </b>
      <p
        class="favorites__status-description"
      >
        Save properties to narrow down search or plan your future trips.
      </p>
    </div>
  </section>
</div>
`;
